### DISCUSSION TITLE
New Version

### DISCUSSION THREAD

**Discussion by VentuzTechnology** [VentuzTechnology/Ventuz.Extension.prerelease#26] [Announcements]
In this thread we will inform you whenever we add a new Ventuz installer with fixes, features and changes.

**Comment by VentuzTechnology** [comment]
**Added Build 6.12.00_242_37316**
Download the new version [here](https://github.com/VentuzTechnology/Ventuz.Extension.prerelease#ventuz-versions) and check the 
[Change Log](https://github.com/VentuzTechnology/Ventuz.Extension.prerelease/blob/main/VersionChangeLog.md) for changes.

**All your extensions must be recompiled!**

**Comment by gielians** [comment]
This version does not seem to load any extentions for me 211 and 219 do though....

**Reply by VentuzDanielWiller** [reply]
Oh sure that was the part that I missed to mention, you need to recompile your Extensions, sorry.
